clang/lib/Headers/stdckdint.h

```diff
@@ -21,9 +21,6 @@
 
 /* C23 7.20.1 Defines several macros for performing checked integer arithmetic*/
 
-#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 202311L
-#define __STDC_VERSION_STDCKDINT_H__ 202311L
```

Comment:
This needs to stay, `__STDC_VERSION_STDCKDINT_H__` is defined by C23, not just something Clang invented